jsk_2016_01_baxter_apc/data/astra/stereo_left_astra_mini_8_rgb.yaml

```diff
@@ -0,0 +1,20 @@
+image_width: 640
+image_height: 480
+camera_name: narrow_stereo/left
```

Comment:
fixed in the new commit.
"rgb_Orbbec_Orbbec    " seems default name, but it works so far.